index.js

```diff
@@ -6,30 +6,16 @@ var xtend = require("xtend")
 
 module.exports = createXHR
 createXHR.XMLHttpRequest = window.XMLHttpRequest || noop
-createXHR.XDomainRequest = "withCredentials" in (new createXHR.XMLHttpRequest()) ? createXHR.XMLHttpRequest : window.XDomainRequest
 createXHR.qsSerialize = null // Define this as a function to support the `qs` option
 
-forEachArray(["get", "put", "post", "patch", "head", "delete"], function(method) {
+"get,put,post,patch,head,delete".split(",").map(function(method) {
```

Comment:
I think either are probably fine, although I am more used to seeing the array syntax.

If we want to prioritize file size to this extent, then I think there are other low-hanging fruit that would bring even bigger wins; i.e.; an "xhr.modern.js" file that would have zero polyfills (and just use `Object.assign`); that would save [a bit more than 6 characters](https://github.com/Raynos/xtend/blob/82bd7f010f519c55ccc55d6c58a7759125677878/immutable.js).